build.gradle

```diff
@@ -73,6 +74,7 @@ dependencies {
 
     // https://mvnrepository.com/artifact/org.mockito/mockito-core
     testImplementation group: 'org.mockito', name: 'mockito-core', version: '3.11.2'
+    testImplementation group: 'org.apache.commons', name: 'commons-csv', version: '1.9.0'
```

Comment:
I am not sure, but maybe you can use the JSON format instead of the CSV one? I would prefer not to add the libraries without which we can live

Reply:
It'll be the way more better, thanks